services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/validation/CableReportErrorValidation.kt

```diff
@@ -19,7 +19,6 @@ import java.util.Locale
 fun CableReportApplicationData.validateForErrors(): ValidationResult =
     validate { notJustWhitespace(name, "name") }
         .and { notJustWhitespace(workDescription, "workDescription") }
-        .and { notJustWhitespace(customerReference, "customerReference") }
```

Comment:
Why were these checks removed?

Reply:
Oh, they were removed from the class.

Reply:
Correct.

Reply:
I didn't notice at first, because shuffling the fields obfuscated what meaningful changes had been done.